CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# Release 0.14.0-dev
+
+### New features
```

Comment:
Prior, it said, `### New features since last release`. I think I prefer this though. :thinking:

Reply:
Let's leave it as is!